ATACseq_footprinting/atacseq_footprinting_pipeline.sh

```diff
@@ -34,7 +38,26 @@ OUTPUT_DIR=/oak/stanford/groups/smontgom/kameronr/ATACseq/output
 module load samtools/1.9
 
 ##################################################################################################################################
-#####################################---STEP 1: SORT, MERGE, AND INDEX BAM FILES---############################################### 
+#####################################---STEP 1: CREATE COVERAGE BIGWIG TRACK---################################################### 
+##################################################################################################################################
+#get chromosomes available in fasta (fasta chroms) -- because it's needed for making the bigwig tracks.
+#OUTPUT_DIR=/oak/stanford/groups/smontgom/kameronr/ATACseq/test
+#this next line might not work? seems like a python command? https://www.biostars.org/p/173963/
+
+if [ ! -f $genome_folder/chromsizes.bed ]; then
+    samtools faidx mm9_bgzip.fa.gz
+    cut -f1,2 mm9_bgzip.fa.gz.fai > sizes.genome
+
+    awk '{{ print $1, 0, $2 }}' chromsizes.txt > chromsizes.bed
+    echo "creation of BigWig coverage track complete"
+else
+    echo "coverage files for the BigWig track have already been created"
```

Comment:
Are the BigWig track files bed files? I thought bigwig is a separate file format than bed files.